sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSamplerBuilder.java

```diff
@@ -98,7 +98,11 @@ public JaegerRemoteSamplerBuilder setInitialSampler(Sampler initialSampler) {
   /**
    * Sets the managed channel to use when communicating with the backend. Takes precedence over
    * {@link #setEndpoint(String)} if both are called.
+   *
+   * @deprecated Use {@link #setEndpoint(String)}. If you have a use case not satisfied by the
+   *     methods on this builder, please file an issue to let us know what it is.
    */
+  @Deprecated
   public JaegerRemoteSamplerBuilder setChannel(ManagedChannel channel) {
```

Comment:
Should be able to delete this method since this code is more recent than the latest release.

Reply:
This method has been around for a while right? e.g.

https://github.com/open-telemetry/opentelemetry-java/blob/v1.9.0/sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSamplerBuilder.java#L58

Reply:
Oops so it has. Was confusing it with something else.